lib/pymedphys/experimental/pinnacle.py

```diff
@@ -0,0 +1,21 @@
+from pymedphys._vendor.deprecated import deprecated as _deprecated
+from pymedphys.pinnacle import PinnacleExport as _PinnacleExport
+from pymedphys.pinnacle import PinnacleImage as _PinnacleImage
+from pymedphys.pinnacle import PinnaclePlan as _PinnaclePlan
+from pymedphys.pinnacle import export_cli as _export_cli
+
+PinnacleExport = _deprecated(
```

Comment:
Make sure to include the "name" part too.

...I also don't believe we have a deprecated logic for the CLI. Potentially still expose the CLI as it was, and get it to call this deprecated function, that way, the warning will come up with the CLI?

Reply:
Hi @pchlap,

Sorry I didn't respond here. I actually can't remember. ... looks like I needed to leave more breadcrumbs for my future self, and others ...